test/app.spec.js

```diff
@@ -1,20 +0,0 @@
-'use strict'
-
-/* eslint-env jest */
-
-import React from 'react'
-import { shallow } from 'enzyme'
-
-import { App } from 'components/App'
-
-jest.mock('lib/client', () => {
-  const CozyClient = jest.requireActual('cozy-client').default
-  return new CozyClient({})
-})
-
-describe('App component only', () => {
-  it('should be mounted correctly', () => {
-    const component = shallow(<App />).getElement()
-    expect(component).toMatchSnapshot()
```

Comment:
we don't test anymore App.jsx ?

Reply:
No it used enzyme shallow(), we can't do that kind of test with RTL. It's too complex to test App with RTL right now because it's a huge integration test